dallinger/heroku/tools.py

```diff
@@ -52,7 +107,11 @@ def bootstrap(self):
 
         self._run(cmd)
         # Set HOST value
-        self.set("HOST", self.url)
+        self.set_multiple(
```

Comment:
Ideally we'd just have the one, but I can't be certain existing experiments don't make use of this API directly. Additionally, the `set()` spelling allows for variable names with spaces in them (which may well be entirely invalid and useless), and the tests currently test for that.

Reply:
I can definitely replace the existing usages of `set()` within Dallinger though.

Reply:
Interesting.... We should have a large discussion around what constitutes the public dallinger API, and could be expected to remain stable. I think of this as limited to:
* the Flask routes
* the `dallinger` namespace JS API from dallinger2.js
* the SQLAlchemy models
* db session stuff, because it seems unavoidable once you can work with the models
* the CLI (`dallinger [blah]`)

Everything else is fair game for radical change.